src/gosling-track/gosling-track.ts

```diff
@@ -303,8 +315,8 @@ function GoslingTrack(HGC: import('@higlass/types').HGC, ...args: any[]): any {
                 return;
             }
 
-            tile.graphics.clear();
-            tile.graphics.removeChildren();
+            tile.graphics?.clear();
+            tile.graphics?.removeChildren();
```

Comment:
is graphics always defined? if so, I can remove this...

Reply:
I believe they are always defined. Many `mark/*.ts` files also consider `tile.graphics` to be defined.